<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QTabWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Java PKS - Panel Sterowania</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="consoleTab">
   <attribute name="title">
    <string>Konsola</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0" rowspan="6">
     <widget class="QTextBrowser" name="consoleTextBrowser"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Stan łącza</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="stateLabel">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Połącz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="nickLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="placeholderText">
       <string>Podaj nick</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="logButton">
      <property name="text">
       <string>Zakończ</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QGroupBox" name="databaseStateGroupBox">
      <property name="title">
       <string>Stan bazy</string>
      </property>
      <layout class="QVBoxLayout" name="databaseStateVerticalLayout">
       <item>
        <widget class="QLabel" name="databaseSizeLabel">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>143</width>
        <height>123</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="ChatTab">
   <attribute name="title">
    <string>Chat</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextBrowser" name="messagesTextBrowser"/>
    </item>
    <item>
     <widget class="QLineEdit" name="messageLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nickLineEdit</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>consoleTextBrowser</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>messagesTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
